contents/docs/feature-flags/early-access-feature-management.mdx

```diff
@@ -17,7 +23,12 @@ Early access features can be created and edited from the [Early Access Managemen
 
 Features are powered by feature flags. Flags are automatically created when a feature is created (based on the name of the feature), or you can link an existing feature flag to a feature. Each feature can also contain a description and documentation link to help users understand what the feature is for.
 
-![Create feature flags](../../images/features/feature-flags/create-feature-release.png)
+<ProductScreenshot
```

Comment:
I don't think this one is right. It should be a screenshot on https://us.posthog.com/early_access_features or the page after creating one, rather than a feature flag. Maybe this was messed up before.